cli/login_test.go

```diff
@@ -170,6 +171,10 @@ func TestLogin(t *testing.T) {
 
 		pty.ExpectMatch("Paste your token here:")
 		pty.WriteLine(client.SessionToken())
+		if runtime.GOOS != "windows" {
+			// For some reason, the match does not show up on Windows.
+			pty.ExpectMatch(client.SessionToken())
+		}
```

Comment:
Manually validated that this works on Windows, just appears to be a weird feature of ptytest. Disabling the session token echo check on Windows for now.